### PR TITLE
【Hackathon 5th No.19】Update rfcs of ContinuousBernoulli and MultivariateNormal API

### PR DESCRIPTION
**Pull request by NKNaN**
Update rfcs of ContinuousBernoulli and MultivariateNormal API

### CODE REVIEW

File: rfcs/APIs/20230927_api_design_for_continuous_bernoulli.md [hunk 1]
```diff
@@ -653,9 +653,10 @@ class ContinuousBernoulli(distribution.AutoCompositeTensorDistribution):
 
 ## 命名与参数设计
 ```python
-paddle.distribution.continuous_bernoulli(probs)
+paddle.distribution.continuous_bernoulli(probability, eps=0.02)
```
Comment (cxxly) [inline rfcs/APIs/20230927_api_design_for_continuous_bernoulli.md:656]: 和PyTorch签名保持一致

File: rfcs/APIs/20230927_api_design_for_multivariate_normal.md [hunk 2]
```diff
@@ -346,17 +346,19 @@ class MultivariateNormalTriL(
...
 # 四、对比分析
-Pytorch 与 Tensorflow 实现方式大体类似，都是通过基本的概率计算得到相应的概率属性。
+Pytorch 的 `MultivariateNormal` 类支持用户输入 `covariance_matrix` ， `precision_matrix` ， `scale_tril` 其中任意一种矩阵用来表示多元正态分布的方差，类中的计算将输入的任意一种转化为 `scale_tril` 然后进行计算，速度更快。而 Tf 中则是根据不同的输入矩阵类型设计了不同的类，如： `tfp.distributions.MultivariateNormalTriL` ， `tfp.distributions.MultivariateNormalFullCovariance` ， `tfp.distributions.MultivariateNormalDiag` 等。Pytorch 的设计简洁，能够涵盖的情况更广，因此参照 Pytorch 进行本 API 设计。
 
 # 五、设计思路与实现方案
 
 ## 命名与参数设计
 ```python
-paddle.distribution.multivariate_normal(loc, sacle)
+paddle.distribution.multivariate_normal(loc, covariance_matrix=None, precision_matrix=None, scale_tril=None)
```
Comment (cxxly) [inline rfcs/APIs/20230927_api_design_for_multivariate_normal.md:355]: 和PyTorch签名保持一致，class用Pascal写法，`MultivariateNormal(loc, covariance_matrix=None, precision_matrix=None, scale_tril=None)`

File: rfcs/APIs/20230927_api_design_for_continuous_bernoulli.md (hunk 1 above)
Comment (NKNaN) [reply rfcs/APIs/20230927_api_design_for_continuous_bernoulli.md]: 已修改

File: rfcs/APIs/20230927_api_design_for_multivariate_normal.md (hunk 2 above)
Comment (NKNaN) [reply rfcs/APIs/20230927_api_design_for_multivariate_normal.md]: 已修改

File: rfcs/APIs/20230927_api_design_for_ContinuousBernoulli.md [hunk 3]
```diff
@@ -653,9 +653,10 @@ class ContinuousBernoulli(distribution.AutoCompositeTensorDistribution):
 
 ## 命名与参数设计
 ```python
-paddle.distribution.continuous_bernoulli(probs)
+paddle.distribution.ContinuousBernoulli(probs=None, lims=(0.499, 0.501))
```
Comment (cxxly) [inline rfcs/APIs/20230927_api_design_for_ContinuousBernoulli.md:656]: probs=None -> probs, probs是必选参数

Comment (NKNaN) [reply rfcs/APIs/20230927_api_design_for_ContinuousBernoulli.md]: 已修改

**Review by cxxly — APPROVED** [review]
LGTM

**Pull request merged**